mappings/net/minecraft/structure/StructureStart.mapping

```diff
@@ -29,6 +29,8 @@ CLASS net/minecraft/class_3449 net/minecraft/structure/StructureStart
 		ARG 1 world
 		ARG 4 box
 		ARG 5 pos
+	METHOD method_14976 moveInsideHeights (Ljava/util/Random;II)V
+	METHOD method_14978 belowTheSea (ILjava/util/Random;I)V
```

Comment:
What does this do? This is not a verb phrase and the action done is unclear given this name.

Reply:
belowTheSea = below the sea level 
moveInsideHeights = make sure that it is in the heights of the world

Reply:
this is a method that returns void. move inside heights is perfectly clear, but is below the sea level moving below the sea level? if yes call it `moveBelowSeaLevel`

Reply:
https://github.com/FabricMC/yarn/pull/1218/commits/44f804dbd8c9317fd41f33f6130aef6e803a66b6